docs/source/tutorial/08-add-a-details-view.mdx

```diff
@@ -118,19 +118,22 @@ fun LaunchDetails(launchId: String) {
 }
 ```
 
-## Handle protocol errors
+## Handle errors
 
-As you execute your query, two types of errors can happen:
+As you execute your query, different types of errors can happen:
 
-* Protocol errors: connectivity issues, HTTP errors, or JSON parsing errors. In this case, `response.exception` will contain an `ApolloException`, and `response.data` will be null.
-* Application errors. In this case, `response.errors` will contain the application errors and `response.data` might be `null`.
+* Fetch errors: connectivity issues, HTTP errors, JSON parsing errors, etc... In this case, `response.exception` contains an `ApolloException`. Both `response.data` and `response.errors` is null.
+* GraphQL [request errors](https://spec.graphql.org/draft/#request-error): in this case, `response.errors` contains the GraphQL errors. `response.data` is null.
+* GraphQL [field errors](https://spec.graphql.org/draft/#field-error): in this case, `response.errors` contains the GraphQL errors. `response.data` contains partial data.
+
+Let's handle the first two for now: fetch errors and GraphQL request errors. 
 
 First let's create a `LaunchDetailsState` sealed interface that will hold the possible states of the UI:
 
 ```kotlin title="app/src/main/java/com/example/rocketreserver/LaunchDetails.kt"
 private sealed interface LaunchDetailsState {
     object Loading : LaunchDetailsState
-    data class ProtocolError(val exception: ApolloException) : LaunchDetailsState
+    data class Error(val message: String) : LaunchDetailsState
```

Comment:
Reminder that these changes must also be applied to the `final` version of the tutorial.